drivers/ethernet/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ zephyr_sources_ifdef(CONFIG_ETH_SMSC911X	 eth_smsc911x.c)
 zephyr_sources_ifdef(CONFIG_ETH_STM32_HAL	 eth_stm32_hal.c)
 zephyr_sources_ifdef(CONFIG_ETH_LITEETH		 eth_liteeth.c)
 zephyr_sources_ifdef(CONFIG_ETH_W5500		 eth_w5500.c)
+zephyr_sources_ifdef(CONFIG_ETH_LOWRISC      eth_lowRISC.c)
+zephyr_sources_ifdef(CONFIG_ETH_LOWRISC      mdiobb.c)
```

Comment:
I am pretty sure they have a MDIO driver in zephyr - we can investigate this later.